js/evolugrid.js

```diff
@@ -497,10 +497,10 @@
 	    		
 	    		    		
 	    		if (extendedDescriptor.export_csv) {
-	    			var span = $('<span/>').click(function(){$this.evolugrid('csvExport');});
-	    			span.append($('<i/>').addClass('glyphicon glyphicon-file').attr('style', 'cursor:pointer;'));
-	    			span.append("Export to CSV");
-	    			pager.append(span);
+					var button = $('<button class="btn btn-success" type="button">').click(function(){$this.evolugrid('csvExport');});
+					button.append($('<i/>').addClass('glyphicon glyphicon-file').attr('style', 'cursor:pointer;'));
```

Comment:
Shall we really switch to a button?
If yes, maybe we don't need the `attr('style', 'cursor:pointer;')` part any more.